Cargo.toml

```diff
@@ -33,6 +33,7 @@ rand_distr = "0.4.3"
 rayon = { version = "1.8.0", optional = true }
 quickcheck = { version = "1.0.3", optional = true }
 sdl2 = { version = "0.36", optional = true, default-features = false, features = ["bundled"] }
+katexit = "0.1.4"
```

Comment:
Should this not be a `dev-dependency`?

Reply:
Doesn't work, I tried

Reply:
I'd rather not add this as a required dependency for all users. I think this can be avoided by following the approach from https://github.com/CEED/libCEED/blob/main/rust/libceed/Cargo.toml

Reply:
I hope that "katexit" feature will be enabled during the "cargo publish"